shuup/core/migrations/0005_data_migrate_language_fallback.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.apps import apps as django_apps
```

Comment:
@Pikkupomo, the function in which I reference apps, already passes apps as an argument, so there's a conflict. apps passed by the function is different from django.apps.apps